app/models/qdm/attributes/attribute.rb

```diff
@@ -0,0 +1,64 @@
+module QDM
+  # Represents QDM attribute
+  class Attribute
+    include Mongoid::Document
+    embedded_in :data_element
```

Comment:
Why is a QDM::Attribute `embedded_in` a `:data_element`?  `DataElement` does not have an `embeds_one` or `embeds_many` on it.

Reply:
Forgotten Artifact from experimentation -- good catch